src/calls.c

```diff
@@ -57,9 +57,14 @@ void SET_NAME_FUNC(Obj func, Obj name)
     FUNC(func)->name = name;
 }
 
+Obj NAMI_OBJ_FUNC(Obj func, Int i)
+{
+    return ELM_LIST(NAMS_FUNC(func),i);
+}
+
```

Comment:
I'd prefer to not do this, let's instead change NAMI_FUNC, NAME_LVAR, etc to return a GAP string, to fix a ton of additional identical problems not yet covered by this PR.

I just sat down (during vacation, yes...) and had a quick look, it shouldn't be a problem, and no package calls these, either. Might even be able to do this for NAME_RNAM and NameGVar, too